system/Database/Seeder.php

```diff
@@ -126,7 +126,21 @@ public function __construct(Database $config, BaseConnection $db = null)
 
 		$this->db    = &$db;
 		$this->forge = Database::forge($this->DBGroup);
-		$this->faker = class_exists(Factory::class) ? Factory::create() : null;
+	}
+
+	/**
+	 * Gets the Faker Generator instance.
+	 *
+	 * @return \Faker\Generator|null
+	 */
+	public static function faker()
```

Comment:
You change with this:

```suggestion
      public static function faker(): ?\Faker\Generator
```

and run :

```bash
vendor/bin/rector process system/Database/Seeder.php
```

so it should will automatically add use statement for `use Faker\Generator` to use statements. It doesn't sort use statements so probably need manual check or use tools like php-cs-fixer to sort it, eg:

```php
php-cs-fixer fix system/Database/Seeder.php --rules=ordered_imports
```